spark/notebooks/Iceberg - View Support.ipynb

```diff
@@ -0,0 +1,525 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "1041ae6f",
+   "metadata": {},
+   "source": [
+    "![iceberg-logo](https://www.apache.org/logos/res/iceberg/iceberg.png)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6a5c8206",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pyspark.sql import SparkSession\n",
+    "spark = SparkSession.builder.appName(\"Jupyter\").getOrCreate()\n",
+    "\n",
+    "spark"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6f9a9f41",
+   "metadata": {},
+   "source": [
+    "## Load Two Months of NYC Taxi/Limousine Trip Data\n",
+    "\n",
+    "For this notebook, we will use the New York City Taxi and Limousine Commision Trip Record Data that's available on the AWS Open Data Registry. This contains data of trips taken by taxis and for-hire vehicles in New York City. We'll save this into an iceberg table called `taxis`."
```

Comment:
As a fellow non-native English, I also have the habit of using `we` but I don't think it is necessary and removing it makes the message easier to convey:
```suggestion
    "This notebook uses the New York City Taxi and Limousine Commission Trip Record Data available on the AWS Open Data Registry. This contains data of trips taken by taxis and for-hire vehicles in New York City. This data is stored in an iceberg table called `taxis`."
```